Azure monitor: Azure Metrics Batch API Implementation frontend (#123697)
grafana/grafana: docs/sources/datasources/azure-monitor/configure/index.md

Querying Basic Logs tables incurs additional costs on a per-query basis.
{{< /admonition >}}

### Enable Batch API

Toggle **Enable Batch API** to query metrics for multiple resources in a single request using the [Azure Monitor Metrics Batch API](https://learn.microsoft.com/en-us/azure/azure-monitor/metrics/migrate-to-batch-api). Instead of sending one request per resource, Grafana groups eligible metric queries and sends them together. This reduces the number of calls to Azure, which improves query performance and lowers the chance of throttling on dashboards that query many resources.

> Using the Batch API requires the [Monitoring Reader](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/rest-api-walkthrough#retrieve-metric-values-multi-dimensional-api) role at the subscription scope, because requests are sent to the `metrics.monitor.azure.com` data plane endpoint.

Reviewer: ```suggestion
Because requests to the Batch API are sent to the `metrics.monitor.azure.com` data plane endpoint, you need the [Monitoring Reader](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/rest-api-walkthrough#retrieve-metric-values-multi-dimensional-api) role at the subscription scope to use it.
```

{{< admonition type="note" >}}
The Metrics Batch API includes 1,000,000 free API calls per month. Additional calls are billed at $0.01 per 1,000 calls.
{{< /admonition >}}

{{< admonition type="note" >}}

> This setting is only available when the `azureMonitorBatchAPI` feature toggle is enabled.

Reviewer: Is the setting referred to here **Enable Batch API**? If so, I'd move this admonition up nearer where this is mentioned.
